apps/web/src/components/bid-activity/sections/bid-activity-details.tsx

```diff
@@ -22,7 +24,11 @@ export default function BidActivityDetails({
         <h2 className={css.heading}>{content}</h2>
         <p className={css.date}>{date}</p>
       </div>
-      {amount && <p className={css.amount}>{amount}</p>}
+      {amount && (
+        <p className={css.amount}>
+          <Currency value={amount} />
+        </p>
+      )}
```

Comment:
Should this be changed to handle the case of `amount` being `0` (zero)? Or is it intentional to not show this if the amount is zero?

```suggestion
      {typeof amount === 'number' && (
        <p className={css.amount}>
          <Currency value={amount} />
        </p>
      )}
```

Reply:
I'm 99% sure that this is just one of those cases that doesn't matter because in this context (auction bids) there should never be a bid of zero.